policies/project-director-election-process.md

```diff
@@ -0,0 +1,77 @@
+    * Each team representative must share their team's nominations and the reasoning for their nominations first with the candidates, and then, for those candidates who consent to be nominated, with the council's selected facilitator.
+    * The facilitator must publicly share the pool of all nominations and reasonings at least ten days before the final election meeting.
+        * This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting, this does not require restarting the 10-day period.
+
+### Election Meeting
+
+The election meeting is compromised of all members of the Leadership Council who will actively participate in the consensus process as well as the facilitator who will facilitate that process.
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format[^2] (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will share their set of nominees and the reasons for their change if that applies.
+* Facilitator proposes a candidate:
+    * Based on gathered data, the facilitator suggests a candidate they believe will receive the group's consent.
+* Seek objections to the facilitator's proposal
+* Finalize the selection:
+    * Discuss and attempt to resolve any objections, if possible.
+    * Consent has been reached if the proposed choice has no remaining objections! :tada:
+    * If objections persist and the current proposal reaches an impasse, the facilitator should make a new proposal based on the discussion and feedback received.
+    * Continue seeking consensus and addressing objections until a satisfactory candidate is selected.
+        * Note that this process seeks to produce a candidate everyone is satisfied with and has no objections to. Avoid lengthy debates over preferences not related to objections or unmet requirements.
+* Document and announce the selection
+
+### FAQ
+
+#### Why is there no step where we require that people consent to be nominated?
+
+One of the goals of this policy is to create opportunities for new leadership. We want to create an environment that encourages people to step up into power. Sometimes the confidence and faith of a group can give people the confidence they need to consent to being selected. Candidates can always object to accepting a role, but we save that step for last to try to filter out as few candidates as possible who would otherwise have objected prior to hearing all the reasons people have confidence in them.
```

Comment:
I think this should reference that candidates have an opportunity to object before the election meeting.

Reply:
Tried to resolve this in https://github.com/rust-lang/leadership-council/pull/12/commits/75b454c9a7713a873c163ff40953627cdea40389; it's a bit more subtle than just mentioning the election meeting straight up. Instead, I mentioned having the chance to object before public nomination, which is the same thing. If you'd prefer it to be more explicit, please suggest alternative wording; this was the best I could do.